allure-report-plugin/src/main/kotlin/io/qameta/allure/gradle/report/tasks/AllureReport.kt

```diff
@@ -8,13 +8,20 @@ import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.options.Option
+import org.gradle.kotlin.dsl.newInstance
 import org.gradle.kotlin.dsl.property
 import org.gradle.kotlin.dsl.the
+import org.gradle.process.ExecOperations
 import report
 import java.io.File
 import javax.inject.Inject
 
 open class AllureReport @Inject constructor(objects: ObjectFactory) : AllureExecTask(objects) {
+
+    interface Injected {
+        @get:Inject val execOperations: ExecOperations
+    }
```

Comment:
```suggestion
    @get:Inject abstract protected val execOperations: ExecOperations
```

What is the point of adding `interface Injected` interface?
An `abstract protected val` or a constructor argument should be enough.

Reply:
Yep, this was the wrong approach. I had a bunch of uncommited changes but messed with them because started leveraging gradle version and adjusting modules, it became a mess at some point.